### ISSUE TITLE
Property 'retrieve' does not exist on type 'CalculationsResource'

### ISSUE THREAD

**Issue by gwax** [stripe/stripe-node#2153]
### Describe the bug

Typescript type information is missing for the `stripe.tax.calculations.retrieve` api method described here: https://docs.stripe.com/api/tax/calculations/retrieve

### To Reproduce

In IDE, create the following line:

```typescript
const taxCalculation = await stripe.tax.calculations.retrieve(taxCalculationId);
```

### Expected behavior

taxCalculation has type `Stripe.Tax.Calculation | null`

### Code snippets

```JavaScript
const taxCalculation = await stripe.tax.calculations.retrieve(taxCalculationId);
```

### OS

macOS

### Node version

Node v20.12.0

### Library version

stripe v16.6.0

### API version

2024-06-20

**Comment by remi-stripe** [comment]
@gwax This API method just shipped a couple days ago publicly and hasn't yet made it into our SDKs. It should be there in the next version we'll release at the end of the week!

**Comment by xavdid-stripe** [comment]
I forgot to circle back, but this was released the other week in `16.7.0` ([changelog](https://github.com/stripe/stripe-node/blob/master/CHANGELOG.md#1670---2024-08-08)). 

Sorry for the hassle!
